test/e2e/e2e.go

```diff
@@ -2444,3 +2444,77 @@ var _ = ginkgo.Describe("e2e delete databases", func() {
 		//table.Entry("when delete south db on ovnkube-db-2", 2, []string{southDBFileName}),
 	)
 })
+
+var _ = ginkgo.Describe("e2e IGMP validation", func() {
+	const (
+		svcname              string = "igmp-test"
+		ovnNs                string = "ovn-kubernetes"
+		port                 string = "8080"
+		ovnWorkerNode        string = "ovn-worker"
+		ovnWorkerNode2       string = "ovn-worker2"
+		mcastGroup           string = "224.1.1.1"
+		multicastListenerPod string = "multicast-listener-test-pod"
+		multicastSourcePod   string = "multicast-source-test-pod"
+		tcpdumpFileName      string = "tcpdump.txt"
+		retryTimeout                = 5 * time.Minute // polling timeout
+	)
+	var (
+		tcpDumpCommand = []string{"bash", "-c",
+			fmt.Sprintf("apk update; apk add tcpdump ; tcpdump multicast > %s", tcpdumpFileName)}
```

Comment:
Adding tcp dump (or any tool for that matter) like this in every test run is bound to lead to problems... We should look at using tools that are available or look to use an image that has the ones we need (really all we need here is `omping` )  I like to always look at how [Upstream K8's handles this] as a reference (https://github.com/kubernetes/kubernetes/tree/master/test/images)

Reply:
I didn't find the `omping` tool inside the standard `agnhost` image
```go
agnhostImage         = "k8s.gcr.io/e2e-test-images/agnhost:2.26"
```
I tried to look for another image that will contain `omping` , but didn't find either.I grep for it on the Upstream K8's repository without successes (notice all the results are `chomping`/`stomping` and not `omping`):

![image](https://user-images.githubusercontent.com/41959855/122029222-5e5fd900-cdd5-11eb-86ce-5f9e07bdddff.png)

what am I missing ? I’m not sure how to get image that contain the `omping` tool.
Thank!

Reply:
So unfortunately `omping` isn't included in that image... I am checking to see if any of the other upstream k8's images have it.  If not I would make a new image following what was already done in https://github.com/ovn-org/ovn-kubernetes/tree/master/test/e2e/images That specifically contains that `omping` tool, the question then becomes where's the best place to host that image, which I'm not sure about.... ping @trozet

Reply:
After asking around it seems the best way to test the actual multicast dataplane is in fact iperf :) and it looks like the test has already been written in https://github.com/ovn-org/ovn-kubernetes/blob/master/test/e2e/multicast.go#L75-L130

Reply:
The test indeed similar but not quite the same.
As far as I understood, In [this test](https://github.com/ovn-org/ovn-kubernetes/blob/master/test/e2e/multicast.go#L75-L130) we creating multicast listener and multicast source and check if there are multicast traffic between them.
On the other hand, as stated on [this issue](https://github.com/ovn-org/ovn-kubernetes/issues/2089) on this test we want to validate some time after an endpoint endpoint joins a group it will receives IGMP a query from OVN. Therefore, we using `tcpdump` in order to sniff the `IGMP` packet. I did not found a way to sniff this packet only using `iperf` yet.